pkg/insightstore/BUILD.bazel

```diff
@@ -4,31 +4,27 @@ go_library(
     name = "go_default_library",
     srcs = [
         "chunk.go",
-        "filestore.go",
         "filepath.go",
+        "filestore.go",
```

Comment:
nits, should we remove this file from this PR, looks like it should be a part of another PR ( insight feature implementation ) 🤔

Reply:
Sure. This was generated by `make gazelle` but you are right. Let me remove them from this PR.